.github/actions/behavior_test_binding_python/action.yaml

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Test Binding Python
+description: 'Test Core with given setup and service'
+inputs:
+  setup:
+    description: "The setup action for test"
+  service:
+    description: "The service to test"
+  feature:
+    description: "The feature to test"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup
+      shell: bash
+      run: |
+        mkdir -p ./dynamic_test_binding_python &&
+        cat <<EOF >./dynamic_test_binding_python/action.yml
+        runs:
+          using: composite
+          steps:
+          - name: Setup Test
+            uses: ./.github/services/${{ inputs.service }}/${{ inputs.setup }}
+          - uses: actions/setup-python@v4
+            with:
+              python-version: '3.11'
+          - name: Build with maturin
+            shell: bash
+            working-directory: "bindings/python"
+            run: |
+              python -m venv venv
+              source venv/bin/activate
+              pip install maturin[patchelf]
+              maturin develop -E test -F ${{ inputs.feature }}
```

Comment:
It's better to move those part to `Behavior Test Binding Python` instead for better readability and maintaince. Keep those trick part as small as possible.